packages/mobile/android/app/build.gradle

```diff
@@ -163,6 +163,7 @@ android {
         targetSdkVersion rootProject.ext.targetSdkVersion
         versionCode appVersionCode
         versionName "1.18.3"
+        versionName "1.19.0"
```

Comment:
Why do we have two `versionName`s?

Reply:
Oh, thanks, good catch. I'm not sure how this is working, I assume the second one overwrites the first one? I removed it in any case but this might have been a change in the pre-release script 🤔